src/Lean/Parser/Do.lean

```diff
@@ -200,6 +200,12 @@ It should only be used for debugging.
 -/
 @[builtin_doElem_parser] def doAssert    := leading_parser:leadPrec
   "assert! " >> termParser
+/--
+`debug_assert! cond` panics if `cond` evaluates to `false` and the executing code has been built
+with debug assertions enabled (see Lake's `buildType`/`moreLeancArgs`).
```

Comment:
```suggestion
with debug assertions enabled (see the `debugAssertions` option).
```

I suggest pointing them to the option is (from where they can learn of the `buildType` default). Users may wish to enable debug assertions even if they do not want to compile the C code in debug mode.